spec/proxy_spec.lua

```diff
@@ -75,7 +75,7 @@ describe('Proxy', function()
     assert.falsy(proxy:find_service('unknown'))
   end)
 
-  it('does not return old configuration when new one is available', function()
+  pending('does not return old configuration when new one is available', function()
```

Comment:
This needs to be moved to the find service policy spec.

Reply:
I think this one belongs to `configuration_store`. In fact, the same thing it's already tested there. I think we can just delete this test.

Reply:
Yeah. 👍